documentation/user_docs/docs/manual/Manipulating_and_extracting_data_from_SQW_files_and_objects.rst

```diff
@@ -2,166 +2,367 @@
-   - If you specify a vector with two components then the signal will be integrated over that axis between limits specified by the two components of the vector.
-   - Four elements vector define multiple cuts with multiple integration limits in the selected direction. The input
-   would have form ``[plo, rdiff, phi, rwidth]`` This defines an integration axis: minimum range center, distance between range centers, maximum range center, range size for each cut. The number of cuts produced will be the number of ``rdiff`` sized steps between ``plo`` and ``phi``; ``phi`` will be automatically increased such that ``rdiff`` divides ``phi - plo``.
-   For example, ``[106, 4, 113, 2]`` defines the integration range for three cuts, the first cut integrates the axis over ``105-107``, the second over ``109-111`` and the third ``113-115``.
+You may optionally choose to use non-orthogonal axes:
 
+.. code-block:: matlab
 
-**Optional arguments**
+   proj = ortho_proj([1 0 0], [0 1 0], [0 0 1], 'nonorthogonal', true);
 
-::
+This forces the axes to be the ones you define, even if they are not orthogonal in the crystal lattice basis.
 
-   my_cut = cut_sqw (data_source, proj, p1_bin, p2_bin, p3_bin, p4_bin, '-nopix', filename)
+.. warning::
 
+   Plots that are produced plot them as orthogonal axes so any features may be skewed.
 
-- ``'-nopix'`` means that the individual pixel information contributing to the resulting data is NOT retained (at present the default is to retain it, resulting in an output that is an sqw object, whereas using ``'-nopix'`` gives a dnd output).
-- ``filename`` is a string specifying a full filename (including path) for the data to be stored, in addition to being stored in the Matlab workspace.
+   However, it does make reading the location of a feature in a two-dimensional **Q**-**Q** plot straightforward, which is the main reason for doing this.
 
-cut
-===
+Spherical Projections
+---------------------
+
+TBD
+
+Cylindrical Projections
+-----------------------
+
+TBD
+
+Binning arguments
+-----------------
+
+* ``p1_bin``, ``p2_bin``, ``p3_bin`` and ``p4_bin``
+  specify the binning / integration arguments for the Q & Energy axes in
+  the target coordinate system. Each can independently have one of four different forms:
+
+.. warning::
+
+   The meaning of the first, second, third, etc. components changes between each form. Ensure that you have the correct
+   value in each component to ensure your cut is what you expect.
+
+* ``[]``
+  An empty binning range will use the current/previous binning axes in that dimension.
+
+* ``[n]``
+  if a single (scalar) number is given then that axis will be a plot axis and the bin width will be the
+  number you specify. The lower and upper limits are the (current|previous) binning axes in that dimension.
```

Comment:
current|previous - we discussed this. Guessing you mean the bins in the sqw before the cut. Not sure the best way of saying this but let's agree a way of saying this